webpack/assets/javascripts/react_app/components/users/PersonalAccessTokens/PersonalAccessTokensConstants.js

```diff
@@ -4,3 +4,6 @@ export const PERSONAL_ACCESS_TOKENS_FAILURE = 'PERSONAL_ACCESS_TOKENS_FAILURE';
 export const PERSONAL_ACCESS_TOKEN_FORM_SUBMITTED =
   'PERSONAL_ACCESS_TOKEN_FORM_SUBMITTED';
 export const PERSONAL_ACCESS_TOKEN_CLEAR = 'PERSONAL_ACCESS_TOKEN_CLEAR';
+export const MODAL_ID = 'personal-access-tokens-form-modal';
+export const AUTH_CODE_EXAMPLE =
+  'curl -u admin:token http://localhost:5000/api/v2/hosts';
```

Comment:
I don't think a const is good for this since it needs to use the Foreman URL (https://projects.theforeman.org/issues/32373) rather than hardcoded localhost.

Reply:
oops, I thought it was just an example

Reply:
It is an example, but visible to User, so it still should be specific to the foreman instance, rather then to general develop instance.

Reply:
thanks, using it with `foremanUrl` in the component`s JSX